src/app/components/operations/register/register.component.ts

```diff
@@ -0,0 +1,99 @@
+import { tokenReference } from '@angular/compiler';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
import { tokenReference } from "@angular/compiler";
```